src/main/java/org/icatproject/core/manager/GateKeeper.java

```diff
@@ -164,35 +164,102 @@ private void exit() {
 		return publicTables;
 	}
 
-	public List<EntityBaseBean> getReadable(String userId, List<EntityBaseBean> beans, EntityManager manager) {
-
-		if (beans.size() == 0) {
-			return beans;
-		}
-
-		EntityBaseBean object = beans.get(0);
-
-		Class<? extends EntityBaseBean> objectClass = object.getClass();
-		String simpleName = objectClass.getSimpleName();
+	/**
```

Comment:
Add a method description as well, perhaps using some of the description for the return type.